src/ogd/games/LAKELAND/features/TotalSessionTime.py

```diff
@@ -0,0 +1,58 @@
+import json
+from typing import Any, List
+from datetime import timedelta
+from ogd.core.generators.Generator import GeneratorParameters
+from ogd.core.generators.extractors.Feature import Feature
+from ogd.common.models.Event import Event
+from ogd.common.models.enums.ExtractionMode import ExtractionMode
+from ogd.common.models.FeatureData import FeatureData
+
+
+class TotalSessionTime(Feature):
+    def __init__(self, params: GeneratorParameters, threshold: float):
+        super().__init__(params=params)
+        self.threshold = threshold 
+        self.total_time = timedelta(0)
+        self.idle_time = timedelta(0)
+        self.active_time = timedelta(0)
+        self.previous_timestamp = None
+
+    def Subfeatures(self) -> List[str]:
+        return ["TotalTime", "TotalSeconds", "IdleTime", "IdleSeconds", "ActiveTime", "ActiveSeconds"]
```

Comment:
I believe we can remove the "TotalTime" subfeature and the `self.total_time` from the `_getFeatureValues` function. We're string-ifying values on output anyway, so I think the base feature will be identical to what's currently being output as the "TotalTime" subfeature. So we can just make `str(self.total_time)` the base feature.